l1-contracts/contracts/bridgehub/Bridgehub.sol

```diff
@@ -320,6 +323,7 @@ contract Bridgehub is IBridgehub, ReentrancyGuard, Ownable2StepUpgradeable, Paus
 
         bytes32 assetInfo = keccak256(abi.encode(L1_CHAIN_ID, sender, _additionalData));
         ctmAssetIdToAddress[assetInfo] = _assetAddress;
+        ctmAssetIdFromAddress[_assetAddress] = assetInfo;
```

Comment:
it seems like the array is not populated on l1

Reply:
Why not, CTM DT calls this on L1 as well?